src/platforms/native/configuration/transports.mdx

```diff
@@ -45,3 +47,19 @@ int main(void) {
 The transport is invoked in the same thread as the call to
 `sentry_capture_event`. Consider to offload network communication to a
 background thread or thread pool to avoid blocking execution.
+
+## Using an HTTP proxy
+
+The Native SDK allows the configuration of an HTTP proxy through which requests can be  tunneled to our backend. This proxy must allow `CONNECT` requests to establish a proxy connection. It can be configured via the following option interface:
+
```

Comment:
```suggestion

The Native SDK allows the configuration of an HTTP proxy through which requests can be tunneled to our backend. This proxy must allow `CONNECT` requests to establish a proxy connection. It can be configured via the following option interface:

```